### PR TITLE
nsidc-data-access tutorial

### PR DESCRIPTION
**Pull request by egreckase**
creating a place holder for the nsidc data access tutorial

### CODE REVIEW

File: book/_toc.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ parts:
           sections:
             - file: tutorials/git_and_jupyterlab/basic-git
             - file: tutorials/git_and_jupyterlab/jupyter
+            - file: tutorials/nsidc-data-aces
```
Comment (scottyhq) [inline book/_toc.yml]:
```suggestion
            - file: tutorials/nsidc-data-access
```

**Review by scottyhq — CHANGES_REQUESTED** [review]
thanks @egreckase just need to make sure the file name matches in the table of contents

File: book/_toc.yml (hunk 1 above)
Comment (egreckase) [reply book/_toc.yml]: I agree with your suggestion, mine was a typo

File: book/tutorials/nsidc-data-access.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#NSIDC DATA ACCESS Tutorial
+
+#Content Here
```
Comment (scottyhq) [inline book/tutorials/nsidc-data-access.md]:
```suggestion
# NSIDC DATA ACCESS Tutorial

## Content Here
```
Jupyterbook is picky about titles (see https://guidebook.hackweek.io/resources/tutorial-resources.html#formatting). You'll also want to hit the 'resolve conflicts' button on this PR to get the most recent table of contents

**Review by scottyhq — CHANGES_REQUESTED** [review]

**Comment by scottyhq** [comment]
Note the added page is currently under the git/jupyterlab tutorial, so you'll want to move it into its own section in the table of contents:

https://deploy-preview-41--snowex2022.netlify.app/tutorials/nsidc-data-access.html

**Pull request closed without merging**